src/js/profile/mobile/widget/GridView.js

```diff
@@ -299,6 +307,7 @@
 
 				self.on("animationend webkitAnimationEnd", animationEndCallback);
 
+				self.on("change", self._shadeCheckbox);
 				utilsEvents.on(page, pageEvents.SHOW, self._onSetGridStyle);
```

Comment:
Please add `self.off("change", self._shadeCheckbox);` in method `_unbindEvents`

Reply:
Done in LINE334